components/vendors/VendorForm.tsx

```diff
@@ -141,11 +200,14 @@ const VendorForm = ({ vendor, host, onSuccess, onCancel, isModal }: VendorFormPr
     initialValues.vendorInfo['otherTaxType'] = initialValues.vendorInfo?.taxType;
     initialValues.vendorInfo.taxType = 'OTHER';
   }
+  if (vendor.payoutMethods?.length > 0) {
```

Comment:
This crashes the "new vendor" form. We need to:

```suggestion
  if (vendor?.payoutMethods?.length > 0) {
```